gui/src/2D/controls/control.ts

```diff
@@ -1833,6 +1837,20 @@ export class Control {
             return true;
         }
 
+        if (type === PointerEventTypes.POINTERWHEEL) {
+            if (this._host._lastControlOver[pointerId]) {
+                var control = this._host._lastControlOver[pointerId];
+                while (control.parent && control.parent.getClassName() !== "ScrollViewer") {
```

Comment:
We cannot have specific code here. This needs to be overload by the ScrollViewer itself

Reply:
added _onWheelScroll function instead. No more while loop, yay!